corehq/apps/case_importer/do_import.py

```diff
@@ -278,6 +278,8 @@ def __init__(self, search_id, fields_to_update, config, domain, user_id, owner_a
         self.parent_external_id = fields_to_update.pop('parent_external_id', None)
         self.parent_type = fields_to_update.pop('parent_type', self.config.case_type)
         self.parent_ref = fields_to_update.pop('parent_ref', 'parent')
+        self.parent_relationship_type = fields_to_update.pop('parent_relationship_type', 'child')
+        self.parent_identifier = fields_to_update.pop('parent_identifier', None)
```

Comment:
what is the difference between this field and `parent_ref`? Can we use `parent_ref` for all cases?

Reply:
There was quite a lot of back-n-forth on this point already on the ticket.

The summary is that `parent_ref` was a very custom column and is deprecated, so unless we know all of its use cases, it's not something that we should use. So, we ended up adding the new column. For more, do checkout https://dimagi-dev.atlassian.net/browse/USH-563?focusedCommentId=110879

Reply:
offlined with @orangejenny and we believe the conclusion there was incorrect so went and merged these columns (using the new name). If any projects were using the deprecated column they can now use the new one

Reply:
Thanks Cal and Jenny.